File: Assignment 8/Problem5.java
//Accept N number from user and print elements which are multiples of 11


import java.lang.*;
import java.util.*;

class Number
{
	public void Display(int Brr[])
	{
		System.out.println("Elements are :");
		
		for(int iCnt=0;iCnt<Brr.length;iCnt++)
		{
			if((Brr[iCnt]%11)==0 )
			{
				System.out.println(Brr[iCnt]);
			}

		}
	}
}
class Problem5
{
	
	public static void main(String arr[])
	{
		Number nobj= new Number();
		
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter Number of Elements");
		int iSize = sobj.nextInt();
		
		int Arr[]= new int[iSize];
		
		System.out.println("Enter Value of Array : ");
		for(int iCnt=0;iCnt<Arr.length;iCnt++)
		{
			Arr[iCnt]= sobj.nextInt();
		}
		
		nobj.Display(Arr);
	}
}

File: Assignment 11/Problem5.c
//Accept N number from user and Display Summation of digits of each number;

#include<stdio.h>
#include<stdlib.h>

void DigitSum(int Arr[], int iLength)
{
    int iCnt = 0;
	int iSum=0;
	int iDigit=0;
  
    for(iCnt =0; iCnt<iLength ; iCnt++)
    {	
			
            while(Arr[iCnt]!=0)
			{
				iDigit=Arr[iCnt]%10;
				
				iSum=iSum+iDigit;
				
				Arr[iCnt]=Arr[iCnt]/10;
			}
              printf("%d\t",iSum);
			  
			  iSum=0;
    }
}

int main()
{
    int iSize = 0, iCnt = 0;
    int *ptr = NULL;   


    printf("Enter number of elements\n");
    scanf("%d",&iSize);

    ptr = (int *)malloc(sizeof(int) * iSize);

    printf("Enter the values\n");
	
    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
		printf("Enter element %d : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }
	
	DigitSum(ptr, iSize);

	
    return 0;
}